Tests/UInt128Tests/UInt128Tests.swift

```diff
@@ -492,8 +492,10 @@ class BinaryIntegerTests : XCTestCase {
         let lowerBits = UInt64("100000000000000000000000000000001", radix: 2)!
         let upperBits = UInt64("100000000000000000000000000000001", radix: 2)!
         let testResult = UInt128(upperBits: upperBits, lowerBits: lowerBits)
+        let words = testResult.words
 
-        testResult.words.forEach { (currentWord) in
+        XCTAssertEqual(words.count, 2)
```

Comment:
We can’t guarantee that the word count will be 2, as different architectures could have different word sizes. I like the idea of checking the count, but it will have to be derived by diving 128 by whatever the platform’s word size is.